src/presenters/open-graph/description-presenter.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Presenter class for the OpenGraph site name.
+ *
+ * @package Yoast\YoastSEO\Presenters\Open_Graph
+ */
+
+namespace Yoast\WP\Free\Presenters\Open_Graph;
+
+use Yoast\WP\Free\Presentations\Indexable_Presentation;
+use Yoast\WP\Free\Presenters\Abstract_Indexable_Presenter;
+
+/**
+ * Class Description_Presenter
+ */
+class Description_Presenter extends Abstract_Indexable_Presenter {
+
+	/**
+	 * Returns the site name.
+	 *
+	 * @param Indexable_Presentation $presentation The presentation of an indexable.
+	 *
+	 * @return string The title tag.
+	 */
+	public function present( Indexable_Presentation $presentation ) {
+		$description = $this->filter( $this->replace_vars( $presentation->og_description, $presentation ) );
+
+		if ( is_string( $description ) && $description !== '' ) {
```

Comment:
`is_string` check is not needed because you are typecasting the return value in `$this->filter`. You can remove the check here or remove the typecase in the filter method. I prefer the last one.

Reply:
Nice :)